base/applications/msconfig/autochkpage.c

```diff
@@ -0,0 +1,230 @@
+/*
+ * PROJECT:     ReactOS Applications
+ * LICENSE:     LGPL - See COPYING in the top level directory
+ * PURPOSE:     Auto Check (autochk) page handler
+ * COPYRIGHT:   Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+/* INCLUDES ******************************************************************/
+
+#include "precomp.h"
+#include "resource.h"
+#include "autochkpage.h"
+
+/* VARIABLES *******************************************************************/
+
+HWND hAutoChkPage;
+HWND hAutoChkDialog;
+WCHAR Program[] = L"System32\\autochk.exe";
+WCHAR SubKey[] = L"SYSTEM\\CurrentControlSet\\Control\\Session Manager";
+WCHAR ValueName[] = L"BootExecute";
+WCHAR EnableValue[] = L"autocheck autochk *\0";
+
+/* FUNCTIONS *******************************************************************/
+
+INT_PTR CALLBACK
+AutoChkPageWndProc(HWND hDlg, UINT message, WPARAM wParam, LPARAM lParam)
+{
+    HKEY hKey;
+    LONG lResult;
+    WCHAR szValueData[255];
+    DWORD dwcbData = sizeof(szValueData);
+    DWORD dwBufferSize = 0;
+    LPWSTR lpsszMultiString = NULL;
+    size_t szSize;
```

Comment:
`sz` is a confusing prefix since it's already used for strings. Just `size` would work.